src/backend/src/routes/onboarding.routes.ts

```diff
@@ -33,6 +33,17 @@ onboardingRouter.post(
   OnboardingController.createChecklistItem
 );
 
+onboardingRouter.post(
+  '/checklist/item/:checklistItemId/update',
+  nonEmptyString(body('name')),
+  nonEmptyString(body('checklistId')),
+  nonEmptyString(body('description').optional()),
+  nonEmptyString(body('parentChecklistItemId').optional()),
+  body('subtasks').isArray,
```

Comment:
This is my fault but I think instead of subtasks, this should take in subtasksIds and then you can also add the line `nonEmptyString(body('subtaskIds.*'))` to ensure every subtaskId you pass into the array is a nonempty string.

Then in the service method you should check that every subtaskId is a valid subtask